Homework3/task_4.py

```diff
@@ -0,0 +1,14 @@
+import random
+numbers = random.randrange(1,14)
+color = random.choice(['clubs', 'diamonds', 'hearts', 'spades'])
+
+if numbers == 1:
+    print("A", color)
+elif numbers == 11:
+    print("J", color)
+elif numbers == 12:
+    print("Q", color)
+elif numbers == 13:
+    print("K", color)
+else:
+    print(numbers, color)
```

Comment:
სწორია და მუშაობს როგორც ჩანს, უბრალოდ ასეც შეგეძლო, მაინც რანდომულად იქნება და ifების წერა ვფიქრობ არ უნდა:
import random

suits = ['♣', '♦', '♥', '♠']
values = ['A', 'K', 'Q', 'J', '10', '9', '8', '7', '6', '5', '4', '3', '2']

# Getting random suit and value
random_suit = random.choice(suits)
random_value = random.choice(values)

print(f"It's: {random_value}{random_suit}")